actors/evm/src/interpreter/precompiles/fvm.rs

```diff
@@ -183,16 +190,19 @@ pub(super) fn call_actor_shared<RT: Runtime>(
         // positive values are user/actor errors
         // success is 0
         let (exit_code, data) = match result {
-            Err(mut ae) => {
-                // TODO handle revert
-                // TODO https://github.com/filecoin-project/ref-fvm/issues/1020
```

Comment:
This now implements changes that closed https://github.com/filecoin-project/ref-fvm/issues/1020